demos/form/form-section.php

```diff
@@ -26,93 +28,93 @@
 
 ////////////////////////////////
 
-$f = \atk4\ui\Form::addTo($app);
-$f->setModel($m, false);
+$form = Form::addTo($app);
+$form->setModel($m, false);
 
-$sub_layout = $f->layout->addSubLayout(\atk4\ui\FormLayout\Section\Generic::class);
+$sub_layout = $form->layout->addSubLayout(Form\Layout\Section::class);
 
 \atk4\ui\Header::addTo($sub_layout, ['Column Section in Form']);
 $sub_layout->setModel($m, ['name']);
 
-$cols_layout = $f->layout->addSubLayout(\atk4\ui\FormLayout\Section\Columns::class);
+$cols_layout = $form->layout->addSubLayout(Form\Layout\Section\Columns::class);
 
 $c1 = $cols_layout->addColumn();
 $c1->setModel($m, ['iso', 'iso3']);
 
 $c2 = $cols_layout->addColumn();
 $c2->setModel($m, ['numcode'/*, 'phonecode'*/]);
 
-$f->addField('phonecode');
+$form->addField('phonecode');
```

Comment:
check `(add|get)(field|element)` matches

after I unsupressed the deprecation output, there are a lot of warnings

but warnings are probably not enought to check, as not all callbacks are triggered in tests